src/frontend/src/pages/Components.tsx

```diff
@@ -1,5 +1,54 @@
+import Button from "components/Button/Button";
+import StartIcon from "assets/icons/star.svg?react";
+import ArrowLeftIcon from "assets/icons/arrow-left.svg?react";
+import ArrowRightIcon from "assets/icons/arrow-right.svg?react";
+
 const Components = () => {
-    return <h1>Components</h1>;
+    return (
+        <div className="bg-gray-50 min-h-screen p-10 font-sans">
+            <div className="max-w-4xl mx-auto space-y-8">
+                <header className="space-y-2">
+                    <h1 className="text-4xl font-bold text-gray-900">
+                        Components
+                    </h1>
+                    <p className="text-gray-600">
+                        A showcase of reusable components built with React Aria
+                        and a custom theme using Tailwind CSS.
+                    </p>
+                </header>
+
+                {/* Section for Buttons */}
+                <section className="p-8 bg-white border border-gray-200 rounded-lg shadow-sm">
+                    <h2 className="text-3xl font-medium text-gray-800 mb-8 pb-4 border-b">
+                        Buttons
+                    </h2>
+
+                    <div className="space-y-8">
+                        <div className="flex flex-wrap items-center gap-4">
+                            <Button variant="primary">
+                                Create your profile
+                            </Button>
+                            <Button variant="outline">
+                                List all neighborhoods
+                            </Button>
+                            <Button variant="outline">
+                                <ArrowLeftIcon className="font-normal h-[18px] w-[18px] text-black" />
```

Comment:
I saw you took out the icon sizing in the button styles but I imagine this group of icon class names would be repeated often, do you feel like it's worth keeping out for now? Looks like this would also be handled in your work for https://github.com/azavea/echo-locator/issues/648 so feel free to ignore!

Reply:
https://github.com/azavea/echo-locator/pull/667/commits/6a32ba2268f52dad8a7edd90933615987e72ab92

Reply:
Thanks for clarifying I misinterpreted that icon size config thinking it affected child component sizing like `ArrowLeftIcon` and `ArrowRightIcon`. I think having a separate icon variant makes sense though!